src/components/pages/Collect/MethodsFilterDropDown.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react'
+import PropTypes, { string } from 'prop-types'
+import OutlinedInput from '@mui/material/OutlinedInput'
```

Comment:
not sure why these warnings crop up when `@mui/material` is indeed listed in the `package json` dependencies. Maybe it's getting confused cause we have two?